Justfile

```diff
@@ -38,38 +36,39 @@ build: build-cli
 # ---------------------------------------------------------------------------- #
 
 test-pkgdb: build-pkgdb
-    @pushd pkgdb; make -j tests; popd
-    @pushd pkgdb; make check; popd
-
-test-env-builder: build-env-builder
-    @pushd env-builder; make -j tests; popd
-    @pushd env-builder; make check; popd
+    @make -C pkgdb -j tests;
+    @make -C pkgdb check;
 
 # Run the end-to-end test suite
 functional-tests +bats_args="": build
-    @flox-tests {{bats_args}}
+    @flox-tests {{bats_args}};
```

Comment:
I think we should start using `shfmt`, but until then I'd prefer if we don't fight over `;` in diffs